src/Features/Core/Portable/DocumentSpanExtensions.cs

```diff
@@ -24,7 +24,7 @@ public static bool TryNavigateTo(this DocumentSpan documentSpan, NavigationBehav
             var service = workspace.Services.GetService<IDocumentNavigationService>();
 
             var options = solution.Options.WithChangedOption(NavigationOptions.PreferProvisionalTab, navigationBehavior != NavigationBehavior.Normal);
-            options = options.WithChangedOption(NavigationOptions.ActivateProvisionalTab, navigationBehavior == NavigationBehavior.PreviewWithFocus);
+            options = options.WithChangedOption(NavigationOptions.ActivateTab, navigationBehavior == NavigationBehavior.PreviewWithFocus);
```

Comment:
There's a slight gap with the `Navigationbehavior` enum now. We've moved to accepting 4 states but it only has 3 values. Probably needs to be updated to 

```csharp
enum NavigationBehavior
{
    NormalWithFocus,
    NormalWithoutFocus,
    PreviewWithFocus,
    PreviewWithoutFocus
}
```

Or changed to a data type with two bools

Reply:
`NormalWithoutFocus` doesn't exist as a use case for us, but we now technically are cool with it now :)

Reply:
Good point. I think I'll get rid of `NavigationBehavior` and just use two bools so the navigation code is consistent throughout.